packages/table/src/table.tsx

```diff
@@ -730,6 +730,13 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
             newSelectedRegions,
         );
 
+        if (
```

Comment:
nit: can we put this up on line 713 to co-locate it with the code that updates `newColumnWidths` and `newRowHeights`?

Reply:
done